src/xcode/ENA/ENA/Source/Scenes/OptionGroup/DatePickerOption/DatePickerDayView.swift

```diff
@@ -0,0 +1,99 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+import Combine
+
+class DatePickerDayView: UIView {
+
+	// MARK: - Init
+
+	@available(*, unavailable)
+	override init(frame: CGRect) {
+		fatalError("init(frame:) has not been implemented")
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	init(viewModel: DatePickerDayViewModel) {
+		self.viewModel = viewModel
+
+		super.init(frame: .zero)
+
+		setUp()
+	}
+
+	// MARK: - Overrides
+
+	override func layoutSubviews() {
+		super.layoutSubviews()
+
+		layer.cornerRadius = bounds.width / 2
+	}
+
+	// MARK: - Private
+
+	private let viewModel: DatePickerDayViewModel
+
+	private var subscriptions = [AnyCancellable]()
+
+	private let titleLabel = DynamicTypeLabel()
```

Comment:
Could you please check whether using ENALabel would make your life easier here? That class has a lot of pre-defined UI looks that were checked with the designers - perhaps they used one of the already existing design elements, then you could just reuse it.

Reply:
In general, double-checking whether there are labels that could be ENALabels would be grand. 👍

Reply:
I checked the ENALabel styles before, but the date picker uses different font size and weight combinations than we usually do so I couldn't use any of the existing styles. That's why I directly use the DynamicTypeLabel here. But I'm aware that we should use them wherever possible. :)

Reply:
Is there the option to include this style into the ENALabel configuration, as an additional case? Or is this particular size / weight / color combination never going to be used again? Just asking, if it is a small change, including it into ENALabel seems to be more consistent :-) But I appreciate that you are aware of the original class, I just wanted to double check 💯

Reply:
I guess that they probably won't be used again. It would be three new styles that are currently only used in this component. And as I see it, we currently partially mirror UIFont.TextStyle in our ENALabel, so these new styles would fall out of that concept. I would leave it with the DynamicTypeLabel for this special case if you're fine with it 😁

Reply:
That's alright then 💯